### PR TITLE
Create edge_type_comparison_graph for the 3 figures

### PR DESCRIPTION
**Pull request by Nourah-Salem**

### CODE REVIEW

File: graphing_outputs/edge_type_comparison_graph.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import numpy as np 
+import matplotlib.pyplot as plt 
+import pandas as pd
+  
+def edge_count(df_fig1, df_fig2, df_fig3):
+    
+    # define axis
+    cartton1 = pd.read_csv(df_fig1)
```
Comment (lagillenwater) [inline graphing_outputs/edge_type_comparison_graph.py:8]: does it matter that this is cartton not cartoon?

**Review by lagillenwater — APPROVED** [review]
I think this looks good. Do you have an example that you could throw in the slack?

@Nourah-Salem Feel free to merge if you think things look good.

File: graphing_outputs/edge_type_comparison_graph.py (hunk 1 above)
Comment (Nourah-Salem) [reply graphing_outputs/edge_type_comparison_graph.py:8]: Good point
Nope, I think we can name it "figure" to be more general

Comment (Nourah-Salem) [reply graphing_outputs/edge_type_comparison_graph.py]: I have made all the edits, but I just cannot make the merge for my pull for some reason!

**Review by bsantan — APPROVED** [review]

**Pull request merged**